soccer/Processor.hpp

```diff
@@ -177,6 +177,8 @@ class Processor : public QThread {
 
     void setFieldDimensions(const Field_Dimensions& dims);
 
+    const bool isRadioOpen();
```

Comment:
The `const` is in the wrong place here.  As is, it says that the return value shouldn't be modified, but I think we'd prefer if it said that calling the method doesn't modify the `Processor` object.  To do this, change it to `bool isRadioOpen() const;`